frontend/routes/package/source.tsx

```diff
@@ -8,6 +8,7 @@ import { PackageHeader } from "./(_components)/PackageHeader.tsx";
 import { TbFolder, TbSourceCode } from "@preact-icons/tb";
 import { ListDisplay } from "../../components/List.tsx";
 import { scopeIAM } from "../../utils/iam.ts";
+import { format as formatBytes } from "@std/fmt/bytes";
```

Comment:
```suggestion
import { format } from "@std/fmt/bytes";
```

I'd tend to use the name provided by the lib, at least everyone has the same name whatever the project.

Reply:
Yep, you're right but to be more descriptive and don't use a generic/common name pretty used that might cause confusion to the LSP I rather used an alias

Reply:
In this case I would be in favor of using lib name. But open an issue on `denoland/std` (not sure of the repo)

Reply:
No, is not needed ;-;. What I meant is that as it is a generic name, is common to be used, so there might more that one function named `format` from the project or a library.